src/server/modules/user/index.ts

```diff
@@ -20,6 +20,10 @@ type LinkInformationProperties = {
 
 type ShortUrlOperationProperty = UserIdProperty & ShortUrlProperty
 
+type OptionalTagsProperty = {
+  tags?: string[] | undefined
```

Comment:
small nit, redundant `undefined` here.

`tags?: string[] | undefined` is same as `tags?: string[]` because of the optional property `?`


 https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-4.html#exact-optional-property-types---exactoptionalpropertytypes

Reply:
nice catch, will fix this